### ISSUE TITLE
`covscan-client` broken by update to `kobo-0.18.0-1.fc34`

### ISSUE THREAD

**Issue by kdudka** [release-engineering/kobo#151]
After upgrading to `kobo-0.18.0-1.fc34`, `covscan-client` stopped working:
```
Traceback (most recent call last):
  File "/usr/bin/covscan", line 58, in <module>
    sys.exit(main())
  File "/usr/bin/covscan", line 50, in main
    parser.run(args)
  File "/usr/lib/python3.9/site-packages/kobo/cli.py", line 290, in run
    self._load_profile(cmd_kwargs['profile'])
  File "/usr/lib/python3.9/site-packages/kobo/cli.py", line 306, in _load_profile
    self.container.conf.load_from_file(configuration_file)
  File "/usr/lib/python3.9/site-packages/kobo/conf.py", line 113, in load_from_file
    fo = open(file_name, "r")
FileNotFoundError: [Errno 2] No such file or directory: '/etc/.conf'
```

After downgrading to `kobo-0.17.0-1.fc34`, `covscan-client` works again.  The failure seems to be related to the following upstream pull request: https://github.com/release-engineering/kobo/pull/148

**Comment by kdudka** [comment]
Could it be caused by the fact that some covscan commands take their own `--profile` option?

**Comment by kdudka** [comment]
The change proposed in #154 fixes the bug for me.

**Comment by rohanpm** [comment]
Released in kobo 0.19.0.
